### ISSUE TITLE
Recorded file name standard

### ISSUE THREAD

**Issue by pikachu937** [bluenviron/mediamtx#2547]
## Describe the feature

I propose to expand the number of parameters for saved files.
1 - the ability to add **%s** (unixtimestamp) to the parameter name, for example: **path1-1698052221.mp4**
2 - the ability to add a file serial number for the current day, for example: **path1-1698052221-000.mp4**; Every new day the serial number is reset to 000.

**Comment by Marco-hub-117** [comment]
I agree, it would be great for me to save the file using %s (unixtimestamp) as the parameter name

**Comment by keidi-xhafa** [comment]
Bumping for %s as parameter for filename

**Comment by aler9** [comment]
The ability to add unix epoch (%s) to recording segment names has been implemented some time ago.
The serial number feature will not be implemented.
